### ISSUE TITLE
plazi treatment matcher fails with due to commons-io library conflict

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/nomer#66]
when running Nomer v0.2.8 - 

[main] ERROR org.globalbioticinteractions.nomer.cmd.CmdLine - org.apache.commons
.io.input.CloseShieldInputStream.wrap(Ljava/io/InputStream;)Lorg/apache/commons/
io/input/CloseShieldInputStream;
java.lang.NoSuchMethodError: org.apache.commons.io.input.CloseShieldInputStream.
wrap(Ljava/io/InputStream;)Lorg/apache/commons/io/input/CloseShieldInputStream;
        at org.globalbioticinteractions.nomer.match.PlaziService.indexTreatments
(PlaziService.java:160)
        at org.globalbioticinteractions.nomer.match.PlaziService.lazyInit(PlaziS
ervice.java:116)

A glance at the dependency tree shows both commons-io versions 2.7 and 2.10 present.

Suspected root cause - a commons-io library conflict.

**Comment by jhpoelen** [comment]
fixed in v0.2.9

**Comment by jhpoelen** [comment]
using nomer v0.2.9 - 

```
$ echo -e "\tHomo sapiens" | nomer append plazi
	Homo sapiens	SAME_AS	http://taxon-concept.plazi.org/id/Animalia/Homo_sapiens_Linnaeus_1758	Homo sapiens	species		Animalia | Chordata | Mammalia | Primates | Hominidae | Homo | Homo sapiens		kingdom | phylum | class | order | family | genus | species	http://taxon-concept.plazi.org/id/Animalia/Homo_sapiens_Linnaeus_1758	
	Homo sapiens	SAME_AS	http://treatment.plazi.org/id/34AC185C73C41EA124EEED97C898FBC0	http://treatment.plazi.org/id/34AC185C73C41EA124EEED97C898FBC0			http://treatment.plazi.org/id/34AC185C73C41EA124EEED97C898FBC0			http://treatment.plazi.org/id/34AC185C73C41EA124EEED97C898FBC0	
	Homo sapiens	SAME_AS	doi:10.5281/zenodo.3917332	doi:10.5281/zenodo.3917332			doi:10.5281/zenodo.3917332			https://doi.org/10.5281/zenodo.3917332	
```
